repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -99,6 +102,48 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
   def this(originalResponse: HttpServletResponse, headerMode: ResponseMode, bodyMode: ResponseMode) =
     this(originalResponse, headerMode, bodyMode, originalResponse.getOutputStream)
 
+  override def setStatus(i: Int, s: String): Unit = {
+    message = s
+    super.setStatus(i, s)
```

Comment:
Alright, I'll add in the Exception for things that don't specifically call out that they do nothing.